frontend/javascripts/types/schemas/dataset_view_configuration_defaults.ts

```diff
@@ -62,6 +63,11 @@ export const enforceValidatedDatasetViewConfiguration = (
           existingLayerConfig,
         );
         eliminateErrors(existingLayerConfig, layerErrors, layerConfigDefault);
+        if (layer.category === "segmentation") {
+          delete existingLayerConfig.intensityRange;
+        } else if (existingLayerConfig.intensityRange == null) {
+          existingLayerConfig.intensityRange = getDefaultIntensityRangeOfLayer(dataset, layer.name);
```

Comment:
Why is this always added for color layers? In my tests, I saved a layer configuration of 

```
{
  "color": {},
  "segmentation": {} 
}
```

and when I reloaded the page, the layer configuration was:

```
{
  "color": {
    "intensityRange": [
      0,
      255
    ]
  },
  "segmentation": {}
}
```

This was unexpected.